tests/integration-tests/configs/common/common.yaml

```diff
@@ -228,7 +228,7 @@ efa:
   test_efa.py::test_efa:
     dimensions:
       - regions: ["sa-east-1"]
-        instances: ["c5n.18xlarge"]
+        instances: ["c5n.9xlarge"]
```

Comment:
let's add a comment here explaining why we're using this instance in place of 18xlarge